analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -450,6 +459,399 @@ public void Finally_NestedFinally()
                 "InInnerFinally",
                 "AfterInnerFinally",
                 "AfterOuterFinally");
+
+            ValidateHasOnlyNoExceptionAndUnknownException(validator, "InOuterFinally");
+            ValidateHasOnlyNoExceptionAndUnknownException(validator, "InInnerFinally");
+            ValidateHasOnlyNoExceptionAndUnknownException(validator, "InInnerTry");
+
+            validator.TagStates("AfterInnerFinally").Should().HaveCount(1)
+                     .And.ContainSingle(x => HasNoException(x));
```

Comment:
_Optional nitpicking:_ Most lines like this can be now a single line, if they contain single item.

Reply:
I've changed these in #5750. I wouldn't change them here to avoid rebase problems.